src/briefcase/commands/base.py

```diff
@@ -117,7 +117,7 @@ def __init__(
         console: Console,
         tools: ToolCache = None,
         apps: dict = None,
-        base_path: Path = Path.cwd(),
+        base_path: Path = None,
```

Comment:
This ensures that `Path.cwd()` is evaluated when the instance is instantiated, rather than when the code is imported, which means it's testable for the default case.